locale-source/locale/en/LC_MESSAGES/django.po

```diff
@@ -4580,3 +4581,54 @@ msgstr "Login with social media account"
 #: adhocracy-plus/templates/socialaccount/snippets/provider_list.html:23
 msgid "Login with "
 msgstr "Login with "
+
```

Comment:
Did you leave these in on purpose, so translators can copy from here to the new strings with adhocracy+ replaced? Im asking bc unfortunately transifex also shows commented out strings, so normally we should remove them, so translators dont have extra work with it...

Reply:
Yes, exactly! And I already copied from them for German. We have the other thing in the docs, so will also remember to delete them next time. :)

Reply:
Ha, perfect!